### ISSUE TITLE
Bug: Duplicating a board notifies users who are mentioned in card descriptions

### ISSUE THREAD

**Issue by chenilim** [mattermost/focalboard#2242]
## Steps to reproduce the behavior

1. Create a card, and mention a user (e.g. yourself) in the card with `@username`
2. You receive a notification (as expected)
2. Duplicate the board
3. You receive another notification
4. Duplicate the card
5. You receive another notification

## Expected behavior

Should not notify mentioned users on duplicated boards or duplicated cards.

## Edition and Platform

 - Edition: Mattermost plugin
 - Version: v0.14.0
 - Browser and OS: Chrome on macOS

**Comment by chenilim** [comment]
@wuwinson, do you agree this is a bug? Or is there another expected behavior. Thanks. cc @wiggin77, @asaadmahmood.

**Comment by wuwinson** [comment]
@chenilim Yes, agree that this is a bug. Users who were previously mentioned on a card should not be notified on a duplicated card unless they are explicitly mentioned on the duplicated card again.

**Comment by chenilim** [comment]
Let's decide on the overall functionality, and circle back.
